src/components/ErrorBoundary/index.native.js

```diff
@@ -0,0 +1,16 @@
+import crashlytics from '@react-native-firebase/crashlytics';
+
+import BaseErrorBoundary from './BaseErrorBoundary';
+import Log from '../../libs/Log';
+
+BaseErrorBoundary.defaultProps.logError = (errorMessage, error, errorInfo) => {
+    // Log the error to the server
+    Log.alert(errorMessage, 0, {error: error.message, errorInfo}, false);
+
+    /* On native we also log the error to crashlytics
+    * Since the error was handled we need to manually tell crashlytics about it */
+    crashlytics().log(`errorInfo: ${errorInfo}`);
```

Comment:
Looking at the logs in crashlytics, it looks like this only logs a string.
![image](https://user-images.githubusercontent.com/3981102/121608944-b1b5de00-ca07-11eb-89a6-5f5a748ec2bb.png)

As a comparison, here's what we see if we stringify the errorInfo in crashlytics under `logs`:
![image](https://user-images.githubusercontent.com/3981102/121610276-7a94fc00-ca0a-11eb-92f5-2dd945094301.png)

I think we can just stringify the log we send to crashlytics, even though we're also logging that on the server it'll only help make it easier to tie the error in crashlytics to our logs since the crashlytics stack trace isn't particularly helpful
![image](https://user-images.githubusercontent.com/3981102/121610429-dd869300-ca0a-11eb-9e6f-ec07ee9f75c2.png)